tests/suites/test_suite_cipher.chacha20.data

```diff
@@ -1,6 +1,6 @@
 Decrypt empty buffer
 depends_on:MBEDTLS_CHACHA20_C
-dec_empty_buf:
+dec_empty_buf:MBEDTLS_CIPHER_CHACHA20
```

Comment:
Isn't this redundant with “Encrypt and decrypt 0 bytes”?

Reply:
It seems that what's decrypted in the "Encrypt and decrypt 0 bytes" test is the encrypted 0 byte buffer, not an actually 0 byte buffer, so no, it doesn't appear to be redundant.

Reply:
This is a stream cipher. The encryption of 0 bytes is 0 bytes.

Reply:
Sounds redundant then. I'll remove it.